scrapyd_k8s/log_handler.py

```diff
@@ -0,0 +1,54 @@
+import os
+from kubernetes import client, watch
+import subprocess
+
+def get_last_n_lines(file_path, num_lines_to_check):
+    lines = []
+    try:
+        with open(file_path, 'r') as file:
+            lines = file.readlines()
+        return lines[-num_lines_to_check:]
+    except FileNotFoundError:
+        return []
+
+def concatenate_and_delete_files(main_file_path, temp_file_path):
+    try:
+        subprocess.run(f"cat {temp_file_path} >> {main_file_path}", shell=True, check=True)
```

Comment:
Is there a reason you're doing this in a subprocess, rather than in Python?

Reply:
It seemed like a simple way, but maybe introduces some overhead, I'll use Python with open thing to append a file to an existing one